### PR TITLE
Run tests on IE9 with browserstack

### PR DESCRIPTION
**Pull request by mattlewis92**
You just need to set the `BROWSERSTACK_USERNAME` and `BROWSERSTACK_ACCESS_KEY` environment variables in your CI and all your tests will automagically get run in IE9 :smile:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,18 +61,18 @@
     "parser": "babel-eslint"
   },
   "scripts": {
-    "lint": "node node_modules/standard/bin/cmd.js src/**/*.js",
```
Comment (jmdobry) [inline package.json]: Why did you change these commands? Now users will have to install these 5 libraries globally in order to run the build. I'd prefer devs be able to just clone, `npm install` and `npm run all` without any additional setup.

Comment (mattlewis92) [inline package.json]: Sorry that was force of habit, npm automatically adds node_modules to your path so you don't actually need those libraries installed globally. It also has greater compatibility for Windows developers

Comment (jmdobry) [inline package.json]: I didn't know that. Do all versions of npm support this?

Comment (jmdobry) [inline package.json]: Will it prioritize `node_modules` binaries over global ones? K now I'm googling it.

File: package.json [hunk 2]
```diff
@@ -60,18 +61,18 @@
...
-    "bundle": "node node_modules/webpack/bin/webpack.js --config webpack.config.js --colors",
-    "min": "node node_modules/uglify-js/bin/uglifyjs dist/js-data.js -o dist/js-data.min.js --source-map dist/js-data.min.map -v -m -c --screw-ie8",
+    "lint": "standard src/**/*.js",
+    "bundle": "webpack --config webpack.config.js --colors",
+    "min": "uglify-js dist/js-data.js -o dist/js-data.min.js --source-map dist/js-data.min.map -v -m -c --screw-ie8",
```
Comment (jmdobry) [inline package.json]: The binary is actually `uglifyjs`

File: package.json (hunk 1 above)
Comment (mattlewis92) [inline package.json]: I think all versions of npm support it, but I'm not 100% sure. `node_modules` gets prioritised over global. More info [here](https://docs.npmjs.com/cli/run-script).

File: package.json (hunk 2 above)
Comment (mattlewis92) [inline package.json]: Oops, will update the PR

**Comment by jmdobry** [comment]
Can you squash into 1 commit?

**Comment by jmdobry** [comment]
1. `git rebase -i HEAD~3`
1. set bottom 2 `pick` values to `s` and save/close
1. set commit message, save/close
1. `git push -f origin karma-browserstack

**Comment by mattlewis92** [comment]
All sorted!

**Comment by jmdobry** [comment]
Hrmm, I think if you drop your changes to "scripts" in package.json, it will let me merge. I already fixed the scripts to not reference node_modules.
